src/main/resources/application-context-schema-org.xml

```diff
@@ -576,6 +576,110 @@
         <property name="converter" ref="solrLongitudeConverter" />
     </bean>
 
+<!-- Extract south bounding coordinate from a 'SO:spatialCoverage' property of type 'GeoCoordinates'. -->
```

Comment:
You added four beans such as `schema_org_geoCoordinates_east`. But they are not referenced by any place.  They should be referenced in the application-context-json-ld.xml file.